app/assets/stylesheets/revised/pages/landing/_home.scss

```diff
@@ -12,18 +12,163 @@
 }
 
 .root-landing-banner-header {
-  background-image: image-url("landing_pages/header_bg_root-1080.jpg");
   width: 100%;
-  padding-top: 12 * $vertical-height;
-  padding-bottom: 6 * $vertical-height;
+  margin-bottom: 0;
+}
+
+.root-landing-banner-subheader {
+  width: 100%;
+  margin: 0;
+
+  .subheader-tile-link {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .subheader-tile:first-of-kind {
+    padding-right: 5px;
+  }
+
+  .subheader-tile:last-of-kind {
+    padding-left: 5px;
+  }
+
+  .subheader-tile {
+    margin: 10px auto;
+    min-width: 275px;
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  .subheader-image {
+    height: 275px;
+  }
+
+  .subheader-image-caption {
+    background-color: black;
+    color: #fff;
+    font-weight: bold;
+    padding: 10px 15px;
+    text-decoration: none;
+  }
+}
+
+.center-navbar-signup-link-container {
+  display: inline-block;
+  width: 75%;
+  text-align: center;
+  margin: 20px auto 10px auto;
+}
+
+.center-navbar-signup-link {
+	background-color: rgba(0,0,0,0);
+	border-radius: 4px;
+	border: 1px solid #3498DB;
+	width: 85px;
+  box-sizing: border-box;
+  color: #fff;
+  cursor: pointer;
+  display: inline;
+  font-size: 0.8rem;
+  font-size: 12px;
+  height: 37px;
+  line-height: 18px;
+  padding: 8px 15px;
+  text-align: center;
+}
+
+@media screen and (min-width: 501px) {
```

Comment:
The collapse actually shows up at `@media (max-width: 991px)`, so this should probably show up then too

Reply:
Updated in d7e17042